### PR TITLE
Deprecated SDK Tools replaced with Command line Tools

### PR DESCRIPTION
**Pull request by koral--**
### Pull Request Checklist

Check/accept these:

- [x] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [x] I've updated the version in the Dockerfile
- [x] I've updated the CHANGELOG.md
- [x] I've provided a link or explanation below which describes the changes in the tool itself
- [x] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added

https://developer.android.com/studio/releases/sdk-tools

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,14 +18,15 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y openjdk-8-jdk libc6:i386 l
...
-    && wget -q https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip -O android-sdk-tools.zip \
-    && unzip -q android-sdk-tools.zip -d ${ANDROID_HOME} \
-    && rm android-sdk-tools.zip
+    && wget -q https://dl.google.com/android/repository/commandlinetools-linux-6514223_latest.zip -O android-commandline-tools.zip \
+    && mkdir -p ${ANDROID_HOME}/cmdline-tools \
+    && unzip -q android-commandline-tools.zip -d ${ANDROID_HOME}/cmdline-tools \
+    && rm android-commandline-tools.zip
 
-ENV PATH ${PATH}:${ANDROID_HOME}/tools:${ANDROID_HOME}/tools/bin:${ANDROID_HOME}/platform-tools
+ENV PATH ${PATH}:${ANDROID_HOME}/tools:${ANDROID_HOME}/tools/bin:${ANDROID_HOME}/platform-tools:${ANDROID_HOME}/cmdline-tools/tools/bin
```
Comment (lmesz-bitrise) [inline Dockerfile:29]: In this case aren't `${ANDROID_HOME}/tools` and and `tools/bin` deprecated and should be removed from the PATH?

File: Dockerfile [hunk 2]
```diff
@@ -18,14 +18,15 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y openjdk-8-jdk libc6:i386 l
...
 # ------------------------------------------------------
-# --- Download Android SDK tools into $ANDROID_HOME
+# --- Download Android Command line Tools into $ANDROID_HOME
 
 RUN cd /opt \
-    && wget -q https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip -O android-sdk-tools.zip \
-    && unzip -q android-sdk-tools.zip -d ${ANDROID_HOME} \
-    && rm android-sdk-tools.zip
+    && wget -q https://dl.google.com/android/repository/commandlinetools-linux-6514223_latest.zip -O android-commandline-tools.zip \
+    && mkdir -p ${ANDROID_HOME}/cmdline-tools \
```
Comment (fehersanyi-bitrise) [inline Dockerfile:25]:
```suggestion
    && mkdir -p ${ANDROID_HOME} \
```
Actually there is no need for this folder, and that way the whole PATH change is unnecessary as well. We should stick to the $ANDROID_HOME/tools/bin structure

Comment (koral--) [reply Dockerfile:25]: Without that you will get error described here: https://stackoverflow.com/questions/60440509/android-command-line-tools-sdkmanager-always-shows-warning-could-not-create-se

File: Dockerfile (hunk 1 above)
Comment (koral--) [reply Dockerfile:29]: true, will change that

**Comment by koral--** [comment]
@lmesz-bitrise @Szabadember @fehersanyi-bitrise @ghaabor could you paste relevant build log part?

**Comment by lmesz-bitrise** [comment]
It could be some intermittent hiccup cause:
`Step 1/35 : FROM quay.io/bitriseio/bitrise-base:alpha
received unexpected HTTP status: 502 Bad Gateway`

**Comment by koral--** [comment]
@lmesz-bitrise @fehersanyi-bitrise is there any update on this?

**Comment by lmesz-bitrise** [comment]
Sorry, I just restarted your build but looks like the problem is around the change.

`Removing intermediate container 311b573b6459
 ---> 102773b2bfd6
Step 7/35 : RUN cd /opt     && wget -q https://dl.google.com/android/repository/commandlinetools-linux-6514223_latest.zip -O android-commandline-tools.zip     && mkdir -p ${ANDROID_HOME}/cmdline-tools/latest     && unzip -q android-commandline-tools.zip -d ${ANDROID_HOME}/cmdline-tools/latest     && rm android-commandline-tools.zip
 ---> Running in fc308a550cbb
Removing intermediate container fc308a550cbb
 ---> d07ead7a30e6
Step 8/35 : ENV PATH ${PATH}:${ANDROID_HOME}/platform-tools:${ANDROID_HOME}/cmdline-tools/latest/tools/bin
 ---> Running in dd454584a577
Removing intermediate container dd454584a577
 ---> 482362ab3643
Step 9/35 : RUN yes | sdkmanager  --licenses
 ---> Running in 0d6621222b0e
Warning: Could not create settings
java.lang.IllegalArgumentException
	at com.android.sdklib.tool.sdkmanager.SdkManagerCliSettings.<init>(SdkManagerCliSettings.java:428)
	at com.android.sdklib.tool.sdkmanager.SdkManagerCliSettings.createSettings(SdkManagerCliSettings.java:152)
	at com.android.sdklib.tool.sdkmanager.SdkManagerCliSettings.createSettings(SdkManagerCliSettings.java:134)
	at com.android.sdklib.tool.sdkmanager.SdkManagerCli.main(SdkManagerCli.java:57)
	at com.android.sdklib.tool.sdkmanager.SdkManagerCli.main(SdkManagerCli.java:48)
Usage:
  sdkmanager [--uninstall] [<common args>] [--package_file=<file>] [<packages>...]
  sdkmanager --update [<common args>]
  sdkmanager --list [<common args>]
  sdkmanager --licenses [<common args>]
  sdkmanager --version`

**Comment by bitrisekristof** [comment]
@koral-- Sorry for the late reply. We are OK with this change but the CI is red and you have to update. :)

**Comment by koral--** [comment]
OK, thanks for the info.

I've first tried to keep `tools` component updating added by @trapacska in https://github.com/bitrise-io/android/commit/e2b0e6e951072d704b14d935d638f009a4521b2a. Now its equivalent is called `commandline-tools;latest`.
We can install `commandline-tools;latest` (or any other fixed version) using `sdkmanager` but it won't be used out of the box because it is installed to a different path (it also contain `/latest/`) which is not in `$PATH`. The latter path cannot be used since the beginning due to this issue: https://stackoverflow.com/questions/60440509/android-command-line-tools-sdkmanager-always-shows-warning-could-not-create-se.
What is more it seems that `tools` update was never effective because the latest release was in September 2017 while linked PR was merged in Novemeber.

So taking that all into account I've just removed SDK tools/Command line tools updating to the latest version at the time of building Docker image.

**Comment by koral--** [comment]
@lmesz-bitrise @fehersanyi-bitrise any update on this?

**Comment by lmesz-bitrise** [comment]
Thank you for the PR and sorry for the delay

**Pull request merged**